### DISCUSSION TITLE
Video downloads no longer possible (any kind of video, any platform)

### DISCUSSION THREAD

**Discussion by ULMOE** [aclap-dev/video-downloadhelper#713] [Help and Q&A] [answered]
### Your question

Hi everyone,
for some time now I can't download any video from whichever platform.
I know I'm not the first one to report this issue. This is not to annoy anyone but to emphazie the urgeny of an answer and preferrably a solution - thank you!

This is just one example of the various **error messages**:
{
  "version": "9.0.1.13",
  "target": "mozilla",
  "channel": "stable",
  "page_url": "https://www.lavita.com/de-de",
  "error": {
    "error": "coapp_failure",
    "details": "SideDownload error: : [tcp @ 0000015324e1bec0] Connection to tcp://customer-powg83vp4off8fm9.cloudflarestream.com:443 failed: Error number -138 occurred\r\n[in#0 @ 000001532644b940] Error opening input: Error number -138 occurred\r\nError opening input file https://customer-powg83vp4off8fm9.cloudflarestream.com/1c3ba0917d4e353b3c7987d87aa34634/manifest/video.mpd?parentOrigin=https%3A%2F%2Fwww.lavita.com.\r\nError opening input files: Error number -138 occurred\r\n",
    "id": "coapp_failure_a7991d65-1db6-4132-9c88-dad081e1611c",
    "downloadable_id": "downloadable_5465280016293844"
  }
}

Here some **examples of videos** that in the past could be downloaded without any problem:
https://youtu.be/IztrZLc3VAM
https://www.lavita.com/de-de (video on the starting page)
https://www.swrfernsehen.de/landesschau-rp/couchgespraeche/erstes-buch-von-shaolin-meister-shi-heng-yi-100.html?mediaId=5d6a0485-1c1a-31a5-81d5-87b402172d7f&

In case any information is missing, please let me know.
Kind regards

### Addon & CoApp version and details

version: 9.0.1.13
target: mozilla
channel: stable
lang: de
coapp: {"found":true,"path":"C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19","new_version":false}
license: {"accepted":true,"email":"<removed>","key":"f80c8aded73c4785"}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:126.0) Gecko/20100101 Firefox/126.0
{
    "valid_license_message_has_been_displayed": true,
    "view_options": {
        "all_tabs": false,
        "low_quality": false,
        "sort_by_status": true,
        "sort_reverse": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "hide_downloaded": false
    },
    "open_count_store": 12
}

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I am just another user.

I am running Windows 11 Pro 64-bit version 23H2 build 22631.3593, Firefox 126.0 64-bit, licensed VDH 9.0.2.3a1 beta, CoApp 2.0.19.

I visited your web site & I was presented with this:

![#01](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/f04350c7-ce29-407a-b684-f70ce3cbc8d6)

I downloaded the item on the VDH menu.  The file I got looked like this:

![#02](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/8ee3f92c-8ff5-4cd7-9447-2e11a8d12ecc)

It played fine in VLC.  While it was playing, VLC told me this:

![#03](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/fc14d157-c5db-4b57-a63b-98f878d5343f)

Although the video played just fine, I couldn't hear any audio.  I cranked my volume up to maximum & still couldn't hear anything.  The Windows file properties claimed there was an audio track.  The VLC properties claimed it was decoding an audio track.  But I couldn't hear anything.  I tried to play the video in the web page.  The top of the page looks like it's the video.  There is a sort of invisible play button lower down in the image.  But it just toggles between play & pause without actually playing anything.  Very weird.

Whatever.  VDH downloaded what was on that page no problem.  Let me quote you:

> for some time now I can't download any video from whichever platform.

For at least 5 years, I have been able to download whatever I wanted from nearly every platform with VDH.  That includes the sample you've provided here.  There have been bugs that prevented downloads.  I won't pretend there haven't been.  But I'm getting the one you've presented here.  I am running the latest beta & you are not.  Maybe it would be worth your while to try the beta.

**Comment by paulrouget** [comment]
- do you use a Proxy
- do you have an antivirus/firewall ? I think your antivirus or firewall blocks the CoApp download request

**Reply by ULMOE** [reply, marked as answer]
C:\Program Files\DownloadHelper CoApp\ffmpeg.exe was blocked by Bitdefender Total Security Firewall.

This was new, as I never had this behaviour before. Must have been introduced by the last update.
After defining an exception rule VDH works as before :-)

Thanks for any support!

**Reply by SeriousConcept1134** [reply]
Thanks for posting this! I've been figuratively ripping my hair out trying to figure this out 😅
